app/styles/_variables.scss

```diff
@@ -45,7 +45,9 @@ $fa-font-path: 'fonts';
 $gradient-percentage: 15%;
 $nav-bar-height: 25px;
 $carousel-control-width: 10%;
-$screen-xs: 400px;
+$screen-xs: 420px;
+$screen-small-tablet: 700px;
+$screen-medium-small: 500px;
```

Comment:
Can we use the [bootstrap breakpoints](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L286) instead of defining our own?

Reply:
i tried but seems to look better at our custom breakpoints.